Bottom: d41bb4ce7402f63336a9806a3ed08231b6f7adca
Top:    f61b8cbd1b4501b444875ed3889515991b0deae4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-21 21:50:50 +0000

Refresh of move-_draw_line-routine-from-d

---

diff --git a/src/draw.c b/src/draw.c
index 8d01922..0e03427 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1226,7 +1226,7 @@ DrawTextLowLevel (hidGC gc, TextType *Text, Coord min_line_width)
 	      newline.Point1.Y += Text->Y;
 	      newline.Point2.X += Text->X;
 	      newline.Point2.Y += Text->Y;
-	      _draw_line (gc, &newline);
+	      gui->graphics->draw_pcb_line (gc, &newline);
 	    }
 
 	  /* move on to next cursor position */
